.travis.yml

```diff
@@ -17,8 +17,6 @@ env:
 
 matrix:
   include:
-    - php: 5.6
```

Comment:
@emteknetnz Are you dropping this here because the build doesn't work, or because we don't generally support <7.1? Either way it's probably best to bump the minimum requirement in `composer.json` at the same time.

Reply:
Updated the composer.json